process-engine/deployment_api_meta
# Fix integration tests

Depends on:
* https://github.com/process-engine/process_engine_contracts/pull/58
* https://github.com/process-engine/process_engine/pull/144
* https://github.com/process-engine/deployment_api_contracts/pull/3
* https://github.com/process-engine/deployment_api_http/pull/2
* https://github.com/process-engine/deployment_api_core/pull/2

## What did you change?

* updated dependencies to feature branches that need to be tested

## How can others test the changes?

All integration tests in meta repositories (including this one) should run successfully: 

* https://github.com/process-engine/management_api_meta/pull/6
* https://github.com/process-engine/consumer_api_meta/pull/40
* https://github.com/process-engine/process_engine_meta/pull/122

## PR-Checklist

Please check the boxes in this list after submitting your PR:
